huntsman/dome/musca.py

```diff
@@ -63,24 +63,33 @@ def __init__(self, *args, **kwargs):
 
     @property
     def is_open(self):
-        v = self._get_shutter_status_dict()[Protocol.SHUTTER]
+        v = self.status[Protocol.SHUTTER]
```

Comment:
`status` is a method, so these will all be, e.g. `v = self.status()[Protocol.SHUTTER]`, which is a bit annoying. This is why we have gone back and forth on it being a property versus a method.